### PR TITLE
fix(DB/SAI) Zul'Farrak: make Antu'sul (8127) despawn his summoned adds (8156) when reset

### PR DESCRIPTION
**Pull request by kjack9**
## Changes Proposed:
-  adds a SAI entry to cause Antu'sul (8127) to despawn his own summoned adds (8156) when reset

## Issues Addressed:

- summoned creatures created by Antu'sul's SAI during a first attempt are not despawned for subsequent attempts

## SOURCE:

- personal experience

## Tests Performed:

- Engaged boss and reset to observe one or more adds being despawned

## How to Test the Changes:

1. `.list creature 8127`
2. `.go creature <entry from above>`
3. target self and `.die`
4. observe that Antu'sul despawns the add he created upon returning to his spawn point

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1684908860029011348.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+--
+-- Antu'Sul (Zul'Farrak)
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1684908860029011348.sql:3]: put always a `DELETE FROM` before `INSERT INTO`
https://www.azerothcore.org/wiki/sql-standards#insert-amp-delete

Comment (kjack9) [reply data/sql/updates/pending_db_world/rev_1684908860029011348.sql:4]: Sorry I didn't review that closer - didn't realize I needed to replace the entire set of SAI entries. Fixed now!

**Review by Helias — APPROVED** [review]

**Comment by kjack9** [comment]
Did anyone have any testing they wanted to do on this before it's merged? Fairly simple change that removes a bug during a simple fight. Let me know if anyone needs anything else to move this forward.

**Comment by Helias** [comment]
@azerothcore/testers

**Pull request merged**